retromock/src/main/java/co/infinum/retromock/Calls.java

```diff
@@ -109,6 +110,11 @@ public Request request() {
       }
       return new Request.Builder().url("http://localhost").build();
     }
+
+    @Override
+    public Timeout timeout() {
+        return new Timeout();
```

Comment:
Let's use Timeout.NONE for now, but this is something we should implement to keep the same behavior as Retrofit.
As we have concept of delay, that is something what timeout should check.